.github/workflows/codeql.yml

```diff
@@ -1,82 +1,17 @@
-# For most projects, this workflow file will not need changing; you simply need
-# to commit it to your repository.
-#
-# You may wish to alter this file to override the set of languages analyzed,
-# or to provide custom queries or build logic.
-#
-# ******** NOTE ********
-# We have attempted to detect the languages in your repository. Please check
-# the `language` matrix defined below to confirm you have the correct set of
-# supported CodeQL languages.
-#
-name: "CodeQL"
+name: "Call CodeQL"
 
 on:
   workflow_dispatch:
   pull_request:
-    types: [opened, synchronize, reopened]
+    types: [ opened, synchronize, reopened ]
   push:
     branches:
       - main
 
-env:
-  JAVA_TOOL_OPTIONS: -Dmaven.wagon.http.retryHandler.class=standard -Dmaven.wagon.http.retryHandler.count=3 -Dmaven.wagon.httpconnectionManager.ttlSeconds=60
 jobs:
-  analyze:
-    name: Analyze
-    runs-on: ubuntu-latest
+  call-codeql:
```

Comment:
rename this job to 'shared-workflows'? Then the text on the status-check (on the PR page) will be:
`Call CodeQL / shared-workflows / Analyze (java) (pull_request`
instead of
`Call CodeQL / call-codeql / Analyze (java) (pull_request` (which is a bit redundant)

Thoughts?